Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM php:7.1
```

Comment:
alpine this bitch up ? cause you know if it's something we run on the fly no one wants to sit there waiting for a 700mb image download